Aptoide/aptoide-client-v8
# [bugfix/MOB-861] Follow Apps and Catappult by default

**What does this PR do?**

   When changing from Realm to Room, there's an issue that if an user upgrades from a version that has Room to a version with Realm, the user stops following the default stores. This fixes it and generalizes the case of updating the app to a specific version.

**Database changed?**

   No


**How should this be manually tested?**

  Install an older version with realm, e.g. 9.13.3.0, and then install this version (it has to count as an update, so it needs to have the same signature). Check that it follows the default stores.

**What are the relevant tickets?**

  [MOB-861](https://aptoide.atlassian.net/browse/MOB-861)


**Code Review Checklist**
